go/vt/sqlparser/sql.y

```diff
@@ -2361,89 +2365,116 @@ column_type_options:
   {
     opt := ColumnType{Null: BoolVal(true), NotNull: BoolVal(false), sawnull: true}
     if err := $1.merge(opt); err != nil {
-    	yylex.Error(err.Error())
-    	return 1
+      yylex.Error(err.Error())
```

Comment:
Probably should preserve the default 4 space indents here and below (better corresponds to the golang tab)

Reply:
All other indents in the file (or at least the majority) use two spaces rather than tabs, so I changed it to be more consistent. Makes sense that we'd use tab instead, but I wanna keep consistency for now.